### PR TITLE
Cleanup Main Part 2

### PR DESCRIPTION
**Pull request by dhenkel92**
This PR is moving everything related to pod/node filter functions into a dedicated function, so that the main is a little bit smaller.
Theoretically, the code was just moved without any modifications.

### CODE REVIEW

File: cmd/draino/draino.go [hunk 1]
```diff
@@ -69,6 +68,96 @@ import (
...
+		"cluster-autoscaler.kubernetes.io/safe-to-evict=false",
+	}
+	pf = append(pf, kubernetes.UnprotectedPodFilter(store, false, append(systemKnownAnnotations, options.protectedPodAnnotations...)...))
+
+	// Cordon Filtering
+	podFilterCordon := []kubernetes.PodFilterFunc{}
+	// TODO do we still need it?
+	// if !options.cordonLocalStoragePods {
+	// podFilterCordon = append(podFilterCordon, kubernetes.LocalStoragePodFilter)
+	// }
```
Comment (dhenkel92) [inline cmd/draino/draino.go:102]: This is a change

File: cmd/draino/draino.go [hunk 2]
```diff
@@ -69,6 +68,96 @@ import (
...
+		}
+	}
+	log.Debug("label expression", zap.Any("expr", options.nodeLabelsExpr))
+	nodeLabelFilterFunc, err := kubernetes.NewNodeLabelFilter(options.nodeLabelsExpr, log)
+	if err != nil {
+		return filtersDefinitions{}, fmt.Errorf("Failed to parse node label expression: %v", err)
+	}
+
+	return filtersDefinitions{
+		// TODO renmae
```
Comment (rrangith) [inline cmd/draino/draino.go:154]: is this still a TODO?

**Review by rrangith — APPROVED** [review]

File: cmd/draino/draino.go (hunk 1 above)
Comment (rrangith) [reply cmd/draino/draino.go:102]: why not delete it? rather than commenting

File: cmd/draino/draino.go (hunk 2 above)
Comment (dhenkel92) [reply cmd/draino/draino.go:154]: Yes, it is.
Basically, draino is not cordoning the nodes anymore, so I think we'll have to remove "cordon" from a several names. But I think it should be done in a separate PR.

File: cmd/draino/draino.go (hunk 1 above)
Comment (dbenque) [reply cmd/draino/draino.go:102]: Historically we have always set the options to `cordonLocalStoragePods` to true.
So we could remove this... but it was in `draino` upstream, so maybe we should keep it for our re-opensource perspectives.
This being said we should rename the option and the filter name: `cordon` does not make sense anymore now. We should go for something like `podFilterCandidate`

**Review by dbenque — APPROVED** [review]
I would keep the podCordonFilter on emptyDir for the moment.

**Pull request merged**